src/ext/java/org/opentripplanner/ext/fares/model/FareLegRule.java

```diff
@@ -0,0 +1,12 @@
+package org.opentripplanner.ext.fares.model;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public record FareLegRule(
+  @Nonnull String feedId,
+  @Nullable String networkId,
```

Comment:
networkId could be `FeedScopedId`

Reply:
The reason that I separated it is that networkId is nullable. Is it allowed to have a null in a FeedScopedId?

Reply:
Ah, that is true